test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -6655,6 +6660,21 @@ def selstat(self, select_list, rattrib, runas=None, extend=None):
         self._disconnect(c)
         return bs
 
+    def skipTest(self, reason=None):
+        """
+        Skip Test
+
+        :param reason: message to indicate why test is skipped
+        :type reason: str or None
+        """
+        if reason:
+            self.logger.warning('test skipped: ' + reason)
+        else:
+            reason = 'unknown'
+        raise SkipTest(reason)
+
+    skip_test = skipTest
```

Comment:
skip_test was just for backward compatibility, we can remove it now, I don't think any tests uses it, + if some test uses it then they will use from PBSTestSuite.skip_test not this one.